frontend/assets/javascripts/src/modules/form/paypal.es6

```diff
@@ -32,14 +32,21 @@ function setupPayment (resolve, reject) {
 
 	if (payment.validateForm()) {
 
+        const checkoutForm = guardian.membership.checkoutForm;
+        const amount = checkoutForm.billingPeriods[checkoutForm.billingPeriod].amount[checkoutForm.currency];
```

Comment:
Is checkoutForm.currency defaulted to something for international (ROW) users? I seem to remember the country code being '--'. (I'm assuming this comes through from the payment details form)